aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -106,17 +106,17 @@ compute_result_type(at::ArrayRef<OperandInfo> operands,
   return compute_result_type(operands, predicates...);
 }
 
-std::tuple<Device, ScalarType> TensorIterator::compute_common_type() {
+static std::tuple<Device, ScalarType> compute_common_type_(at::ArrayRef<OperandInfo> operands) {
```

Comment:
nit: I don't like the underscore at the end, but maybe it's just me. Usually only member variables have an underscore at the end.